tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -1303,24 +1303,25 @@ def state_size(self):
   def output_size(self):
     return self._num_units
 
-  def _norm(self, inp, scope):
+  def _norm(self, inp, scope, dtype=dtypes.float32):
```

Comment:
Do you need a default value here?

Reply:
No, you don't need it but in case some legacy code calls ```_norm``` directly – for whatever reason – the default argument prevents this code from breaking.